src/components/AddIncomeModal/AddIncomeModal.jsx

```diff
@@ -0,0 +1,127 @@
+                Wallet by default (in dollars)
+              </option>
+              <option value="wallet-2">Wallet 2 (in euro)</option>
+              <option value="wallet-3">Wallet 3 (in zl)</option>
+            </Select>
+          </FormControl>
+
+          <FormControl mb="20px" isRequired>
+            <FormLabel>{i18next.t('modal.addIncome.amount')}</FormLabel>
+            <InputGroup>
+              <NumberInput w="100%" precision={2}>
+                <NumberInputField {...register('amount')} />
+              </NumberInput>
+              <InputRightAddon>
+                {i18next.t('modal.addIncome.amount.dollarSign')}
+              </InputRightAddon>
+            </InputGroup>
+          </FormControl>
+
+          <FormControl mb="20px" isRequired>
+            <FormLabel>{i18next.t('modal.addIncome.category')}</FormLabel>
+            <Select
+              placeholder={i18next.t('modal.addIncome.category.placeholder')}
+              {...register('category')}
+            >
+              <option value="category-1">Category 1</option>
+              <option value="category-2">Category 2</option>
+              <option value="category-3">Category 3</option>
+              <option value="category-new">
+                {i18next.t('modal.addIncome.category.option.addNew')}
+              </option>
+            </Select>
+          </FormControl>
+
+          <FormControl mb="20px" isRequired>
+            <FormLabel>{i18next.t('modal.addIncome.date')}</FormLabel>
+            <Input type="date" {...register('date')} />
+            <FormHelperText>
+              {i18next.t('modal.addIncome.date.helperText')}
+            </FormHelperText>
+          </FormControl>
+
+          <FormControl mb="20px">
+            <FormLabel>{i18next.t('modal.addIncome.isRecurring')}</FormLabel>
+            <Select {...register('isRecurring')}>
+              <option value="recurring-no">
+                {i18next.t('modal.addIncome.isRecurring.no')}
+              </option>
+              <option value="recurring-daily">
+                {i18next.t('modal.addIncome.isRecurring.daily')}
+              </option>
+              <option value="recurring-weekly">
+                {i18next.t('modal.addIncome.isRecurring.weekly')}
+              </option>
+              <option value="recurring-monthly">
+                {i18next.t('modal.addIncome.isRecurring.monthly')}
+              </option>
+            </Select>
+          </FormControl>
+
+          <FormControl>
+            <FormLabel>{i18next.t('modal.addIncome.note')}</FormLabel>
+            <Textarea rows={4} {...register('note')} />
+          </FormControl>
+        </ModalBody>
+
+        <ModalFooter>
+          <Button mr="20px" onClick={onClose}>
+            {i18next.t('button.cancel')}
+          </Button>
+          <Button type="submit" onClick={handleSubmit(onSubmit)}>
+            {i18next.t('button.add')}
+          </Button>
```

Comment:
Why did you put buttons in that order?

Reply:
I took this variant from UserStory docs. But I checked it again now, and they added a new version of it :) I will change it to Submit + Cancel
![Screenshot_20220721-181645_Docs.jpg](https://user-images.githubusercontent.com/47990337/180263562-f6837729-5425-4949-8ddc-78d613403227.jpg)
![Screenshot_20220721-181700_Docs.jpg](https://user-images.githubusercontent.com/47990337/180263806-89364213-8fb8-4f68-a165-101725c1ac32.jpg)